xcodeproj/internal/installer.template.sh

```diff
@@ -3,7 +3,7 @@
 set -euo pipefail
 
 readonly src="$PWD/%source_path%"
-readonly dest="${BUILD_WORKSPACE_DIRECTORY}/%output_path%"
+readonly dest="$BUILD_WORKSPACE_DIRECTORY/%output_path%"
```

Comment:
I believe that the curly brace word expansion is the preferred form. Was there a reason to remove it (e.g. template expansion)?

Reply:
It's only needed when you need to disambiguate the name, or do something special like substrings or pattern matching, right?

Here is an example of @keith requesting a similar change in rules_apple: https://github.com/bazelbuild/rules_apple/pull/1417#discussion_r850955300

Reply:
I agree that in this case, it is not necessary. It is more of a style issue, I suppose.

Reply:
Yea I guess we don't really have a specific style for this. My informal style guide for this internally, which I am projecting, is to only use these when it's necessary.